common/metrics/build.rs

```diff
@@ -1,14 +1,17 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
+use std::env;
 use std::process::Command;
 
 /// Save revision info to environment variable
 fn main() {
-    let output = Command::new("git")
-        .args(&["rev-parse", "--short", "HEAD"])
-        .output()
-        .unwrap();
-    let git_rev = String::from_utf8(output.stdout).unwrap();
-    println!("cargo:rustc-env=GIT_REVISION={}", git_rev);
+    if env::var("GIT_REV").is_err() {
+        let output = Command::new("git")
+            .args(&["rev-parse", "--short", "HEAD"])
+            .output()
+            .unwrap();
```

Comment:
Should we try to still have something here(via `unwrap_or`) if neither GIT_REV nor git is there?

For example for cluster test runner I don't really care about git rev of the runner, but because it depends on `common/metrics` through our complex dependency tree, I would still need to pass GIT_REV in order to compile cluster test.

Reply:
Sounds reasonable! updated the changes

Reply:
I think unwrap_or is on the wrong line? If git is missing it will panic on this `unwrap`
You can land it after fix - @bors-libra delegate+

Reply:
@andll It's harder than I thought to use unwrap_or here, because we need to pass a same type which is std::process::Output.

Reply:
I will just set the env for cluster test for now to unblock this

Reply:
I see. Thank you!

For future, one other solution could be using `.ok()` that will convert Result to Option, and then `.map()` to process this option.

So would be something like 
```
let output = ...output().ok(); // Option<process::Output>
let git_rev = output.map(|x|String::from_utf8(x.stdout)).unwrap_or(...);
```

But adding git rev to cluster test works as well